WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+FAB.swift

```diff
@@ -59,15 +59,15 @@ extension MySiteViewController {
         let viewModel = VoiceToContentViewModel(blog: blog) { [weak self] transcription in
             guard let self else { return }
             self.dismiss(animated: true) {
-                // TODO: Are we adding all necessary fields?
                 let presenter = RootViewCoordinator.sharedPresenter
                 let post = blog.createDraftPost()
-                post.content = """
+                let revision = post._createRevision() as! Post
+                revision.content = """
```

Comment:
Based on how the editor works and tracks changes, we have to be pre-populating the revision and not the original.